components/org.wso2.carbon.identity.oauth.stub/src/main/resources/OAuthAdminService.wsdl

```diff
@@ -11,27 +11,27 @@
                     </xs:sequence>
                 </xs:complexType>
             </xs:element>
-            <xs:element name="getAllOAuthApplicationData">
+            <xs:element name="registerOAuthConsumer">
```

Comment:
Can we revert these unnecessary changes and just retain "encryptionAlgorithm" "encryptionMethod" element additions only. It make bit harder to track the changes.